.gitignore

```diff
@@ -99,3 +99,8 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# Apps
```

Comment:
Please add this to you local .gitignore and remove here

Reply:
Done